rhcs5_odf_external/README.md

```diff
@@ -17,10 +17,10 @@ The RHCS cluster will also be deployed on top of the 3 BM nodes using the instal
 
 ## Crucible Automation ##
 
-Crucible automation is a set of playbooks for automating the creation of an OpenShift Container Platform cluster on premise using the Developer Preview version of the OpenShift Assisted Installer.  The key benefits of using crucible automation for our lab environment is it allows us to deploy base Red Hat Openshift 4.9 control planes for both clusters in separate virtual machines across our three physical servers. 
+Crucible automation is a set of playbooks for installing an OpenShift Container platform cluster on premise using the developer preview version of the OpenShift Assisted Installer.  The key benefits of using crucible automation for our lab environment is it allows us to deploy base Red Hat Openshift 4.9 control planes for both clusters in separate virtual machines across our three physical servers. 
```

Comment:
Capitalize the P in platform, that's a proper product name: OpenShift Container Platform.